starter-code/js/index.js

```diff
@@ -3,9 +3,31 @@ var $calc = document.getElementById('calc');
 
 function updateSubtotal($product) {
   // Iteration 1.1
+  let subtot = document.querySelector('.subtot span').innerText;
+  let productPrice = document.querySelector(".pu span").innerText;
+  //console.log(productPrice);
+  let totalQty = document.querySelector('.qty input').value;
+ // console.log(totalQty);
+  let subtotal = productPrice * totalQty;
+  return subtot= subtotal;
+  //console.log(subtot);
+ 
 }
 
 function calculateAll() {
   // Iteration 1.2
-}
-$calc.onclick = calculateAll;
\ No newline at end of file
+  //let prices = document.querySelector(".subtot").innerText;
+  let totalPrice= 0;
+  for (product in cart){
+    //updateSubtotal(product);
+    totalPrice += updateSubtotal(product);
+    }
+  document.querySelector("h2").innerHTML = `<h2>Total: <span>${totalPrice}</span></h2>`;
```

Comment:
You are only going to change the text of the span inside of h2, so you can here use document.querySelector("h2 span") instead. This will target the price directly (the actual number)